public/js/app/templates/sessionNewTemplate.handlebars

```diff
@@ -28,6 +28,9 @@
       {{#if errors}}
         {{#link-to "forgot-password"}}Forgot your password?{{/link-to}}
       {{/if}}
+
+      <p>New to freefeed? <a href="https://docs.google.com/forms/d/1P4cyYUZlxABbWvZrM5jHRcMUFQFpc8qfbZWjGulobsA/viewform?c=0&w=1
+" target="_blank">Create an account »</a></p>
```

Comment:
I'd prefer not use "»" as an arrow. At least we are not doing it anywhere else on the site.